### PR TITLE
Fix/VersionName, Cleanup, Version bump

### PR DESCRIPTION
**Pull request by echobass**

### CODE REVIEW

File: client/src/test/java/com/sendwithus/SendWithUsTest.java [hunk 1]
```diff
@@ -105,6 +106,10 @@ public void testSimpleSend() throws SendWithUsException
     {
         SendReceipt sendReceipt = sendwithusAPI.send(EMAIL_ID,
                 defaultRecipientParams, defaultDataParams);
+        
+        System.out.println(sendReceipt.getEmail().getName());
+        System.out.println(sendReceipt.getEmail().getVersionName());
+        System.out.println(sendReceipt.getEmail().getLocale());
```
Comment (echobass) [inline client/src/test/java/com/sendwithus/SendWithUsTest.java]: Oops

File: client/src/test/java/com/sendwithus/SendWithUsTest.java [hunk 2]
```diff
@@ -441,4 +446,15 @@ public void testConversionEventWithoutRevenue() throws Exception
 
         assertSuccessfulAPIReceipt(receipt);
     }
+    
+    /**
+     * Test deactivate all drips for a customer
+     */
+    @Test
+    public void tests() throws Exception
```
Comment (echobass) [inline client/src/test/java/com/sendwithus/SendWithUsTest.java]: Come on!

File: client/src/main/java/com/sendwithus/model/DeactivatedDripsAll.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.sendwithus.model;
+
+
+public class DeactivatedDripsAll extends APIReceipt {
```
Comment (echobass) [inline client/src/main/java/com/sendwithus/model/DeactivatedDripsAll.java]: Changing class name will break back-compat

Comment (echobass) [inline client/src/main/java/com/sendwithus/model/DeactivatedDripsAll.java]: Done

File: client/src/test/java/com/sendwithus/SendWithUsTest.java (hunk 1 above)
Comment (echobass) [inline client/src/test/java/com/sendwithus/SendWithUsTest.java]: Done

File: client/src/test/java/com/sendwithus/SendWithUsTest.java (hunk 2 above)
Comment (echobass) [inline client/src/test/java/com/sendwithus/SendWithUsTest.java]: Done

File: client/src/test/java/com/sendwithus/SendWithUsTest.java [hunk 4]
```diff
@@ -14,14 +14,15 @@
...
-import com.google.gson.Gson;
 import com.sendwithus.exception.SendWithUsException;
 import com.sendwithus.model.APIReceipt;
 import com.sendwithus.model.APIResponse;
 import com.sendwithus.model.CustomerReceipt;
+import com.sendwithus.model.DeactivatedDrips;
 import com.sendwithus.model.Email;
 import com.sendwithus.model.RenderedTemplate;
 import com.sendwithus.model.SendReceipt;
+import com.sendwithus.model.SendReceipt.EmailSendDetails;
```
Comment (echobass) [inline client/src/test/java/com/sendwithus/SendWithUsTest.java]: Unused import... but we should probably test ;)

File: client/src/main/java/com/sendwithus/model/ActivatedDrip.java [hunk 5]
```diff
@@ -9,11 +9,11 @@
     private String locale; // must match JSON response
     private String message; // must match JSON response
 
-    public DripCampaign getDrip_campaign() {
+    public DripCampaign getDripCampaign() {
```
Comment (demoore) [inline client/src/main/java/com/sendwithus/model/ActivatedDrip.java]: This is all new stuff, right? I'm worried about breaking compatibility.

Comment (echobass) [inline client/src/main/java/com/sendwithus/model/ActivatedDrip.java]: Yup, ActivatedDrip and DeactivatedDrip are both new classes introduced by Yousef.  Nobody should actually be using them yet though, as we haven't published a new maven release

**Pull request merged**